core/citrus-api/src/main/java/org/citrusframework/functions/FunctionParameterHelper.java

```diff
@@ -34,62 +33,94 @@ private FunctionParameterHelper() {}
     
     /**
      * Convert a parameter string to a list of parameters.
-     * 
+     *
      * @param parameterString comma separated parameter string.
      * @return list of parameters.
      */
     public static List<String> getParameterList(String parameterString) {
-        List<String> parameterList = new ArrayList<>();
+        return new ParameterParser(parameterString).parse();
```

Comment:
The longer I think about that... wouldn't it be easier to just use a CSV-Parser for that?

Reply:
Hey @bbortt 
Your thoughts on that?

Reply:
hm.. I think CSV uses double quotes for concatenation. that's a bit problematic in the context of Java. other than that, I do too think that it's the same parsing logic.